lib/static/components/favicon-status.js

```diff
@@ -0,0 +1,76 @@
+'use-strict';
+
+import {PureComponent} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+const DEFAULT = 'default';
+const SUCCESS = 'success';
+const FAILED = 'fail';
+const RUNNING = 'running';
+
+class FaviconStatus extends PureComponent {
+    static propTypes = {
+        status: PropTypes.string
+    }
+
+    link = null;
+
+    getFaviconURL(color) {
+        const canvas = document.createElement('canvas');
+        const diameter = 32;
+
+        canvas.height = diameter;
+        canvas.width = diameter;
+        const ctx = canvas.getContext('2d');
+
+        ctx.beginPath();
+        ctx.arc(diameter / 2, diameter / 2, diameter / 3, 0, 2 * Math.PI);
```

Comment:
If diameter is - `32` why radius - `32 / 3`?

Reply:
In my opinion, `32 / 2` is a large circle size for the `32x32` icon) but if you think it is more preferable I will fix it)

Reply:
I have not tried changing these sizes so I don't know how it looks like with radius `32 / 2`. I don't understand why the radius is so oddly calculated.